### ISSUE TITLE
Website: Wrong Github Page Link

### ISSUE THREAD

**Issue by MentalGear** [crxjs/chrome-extension-tools#197]
### Build tool

Rollup

### Where do you see the problem?

- [X] In the browser
- [ ] In the terminal

### Describe the bug

Instead of leading to the official github page, it leads here: https://github.com/shuding/nextra

### Reproduction

Click top right github icon on: https://www.extend-chrome.dev

### System Info

```shell
Chrome
```

### Severity

annoyance

**Comment by jacksteamdev** [comment]
@MentalGear Thanks for reporting this, I looked into it, and it seems like a problem with the version of Nextra used to build the site. The link is correct in dev mode but wrong when it builds in CI. I tried upgrading Nextra et al. but ran into a rabbit hole of other errors.

I plan to revamp the docsite in Svelte once the Vite version of RPCE is feature complete, so I probably won't fix this now.
